test/algorithms/shoebox/test_mask_overlapping.py

```diff
@@ -105,7 +107,6 @@ def tst_non_overlapping(reflections, non_overlapping, image_size):
 
 def tst_overlapping(reflections, overlapping, adjacency_list, image_size):
```

Comment:
ISTR convention is we now call test functions `test_` not `tst_` - does this mean this test is missed by `pytest`?

Reply:
No, they are helper functions for one main `test()` function. This is pretty old code from pre-pytest days that was just directly converted. Nowadays we would lay it out differently.

Reply:
👍